plugins/generator-1.17.1/forge-1.17.1/mappings/entities.yaml

```diff
@@ -0,0 +1,478 @@
+EntityMarker:
+  - Marker
+  - EntityType.MARKER
+  - marker
+EntityMinecart:
+  - Minecart
+  - EntityType.MINECART
+  - minecart
+EntityMinecartChest:
+  - MinecartChest
+  - EntityType.CHEST_MINECART
+  - chest_minecart
+EntityMinecartCommandBlock:
+  - MinecartCommandBlock
+  - EntityType.COMMAND_BLOCK_MINECART
+  - command_block_minecart
+EntityMinecartContainer: AbstractMinecartContainer
+EntityMinecartFurnace:
+  - MinecartFurnace
+  - EntityType.FURNACE_MINECART
+  - furnace_minecart
+EntityMinecartHopper:
+  - MinecartHopper
+  - EntityType.HOPPER_MINECART
+  - hopper_minecart
+EntityMinecartMobSpawner:
+  - MinecartSpawner
+  - EntityType.SPAWNER_MINECART
+  - spawner_minecart
+EntityMinecartTNT:
+  - MinecartTNT
+  - EntityType.TNT_MINECART
+  - tnt_minecart
+EntityMob: Mob
+EntityMonster: Monster
+EntityMule:
+  - Mule
+  - EntityType.MULE
+  - mule
+EntityMooshroom:
+  - MushroomCow
+  - EntityType.MOOSHROOM
+  - mooshroom
+EntityOcelot:
+  - Ocelot
+  - EntityType.OCELOT
+  - ocelot
+EntityPainting:
+  - Painting
+  - EntityType.PAINTING
+  - painting
+EntityPanda:
+  - Panda
+  - EntityType.PANDA
+  - panda
+EntityParrot:
+  - Parrot
+  - EntityType.PARROT
+  - parrot
+EntityPig:
+  - Pig
+  - EntityType.PIG
+  - pig
+EntityPillager:
+  - Pillager
+  - EntityType.PILLAGER
+  - pillager
+EntityPufferfish:
+  - Pufferfish
+  - EntityType.PUFFERFISH
+  - pufferfish
+EntityPigZombie:
+  - ZombifiedPiglinEntity
+  - EntityType.ZOMBIFIED_PIGLIN
+  - zombified_piglin
+EntityPolarBear:
+  - PolarBear
+  - EntityType.POLAR_BEAR
+  - polar_bear
+EntityTNTPrimed:
+  - PrimedTnt
+  - EntityType.TNT
+  - tnt
+EntityRabbit:
+  - Rabbit
+  - EntityType.RABBIT
+  - rabbit
+EntitySalmon:
+  - Salmon
+  - EntityType.SALMON
+  - salmon
+EntitySheep:
+  - Sheep
+  - EntityType.SHEEP
+  - sheep
+EntityShoulderRiding: ShoulderRidingEntity
+EntityShulker:
+  - Shulker
+  - EntityType.SHULKER
+  - shulker
+EntityShulkerBullet:
+  - ShulkerBullet
+  - EntityType.SHULKER_BULLET
+  - shulker_bullet
+EntitySilverfish:
+  - Silverfish
+  - EntityType.SILVERFISH
+  - silverfish
+EntitySkeleton:
+  - Skeleton
+  - EntityType.SKELETON
+  - skeleton
+EntitySkeletonHorse:
+  - SkeletonHorse
+  - EntityType.SKELETON_HORSE
+  - skeleton_horse
+EntitySlime:
+  - Slime
+  - EntityType.SLIME
+  - slime
+EntitySmallFireball:
+  - SmallFireball
+  - EntityType.SMALL_FIREBALL
+  - small_fireball
+EntitySnowman:
+  - SnowGolem
+  - EntityType.SNOW_GOLEM
+  - snow_golem
+EntitySnowball:
+  - Snowball
+  - EntityType.SNOWBALL
+  - snowball
+EntitySpectralArrow:
+  - SpectralArrow
+  - EntityType.SPECTRAL_ARROW
+  - spectral_arrow
+EntitySpellcasterIllager: SpellcasterIllager
+EntitySpider:
+  - Spider
+  - EntityType.SPIDER
+  - spider
+EntitySquid:
+  - Squid
+  - EntityType.SQUID
+  - squid
+EntityStray:
+  - Stray
+  - EntityType.STRAY
+  - stray
+EntityTraderLlama:
+  - TraderLlama
+  - EntityType.TRADER_LLAMA
+  - trader_llama
+EntityTameable: TamableAnimal

```

Comment:
Possibly `OwnableEntity`

Reply:
From what I've seen `TamableAnimal` should be fine. Maybe we could add more entity types to the datalist in another PR